CHANGES.md

```diff
@@ -4,6 +4,7 @@
 
 - Reduce felix occupancy by replacing endpoint dictionaries by "struct"
   objects.
+- Allow different hosts to have different interface prefixes.
```

Comment:
I think it's worth specifying 'for combined OpenStack and Docker systems' or similar.